shared/hybrid/cordova_plugins.js

```diff
@@ -76,7 +76,7 @@ module.exports = [
 module.exports.metadata = 
 // TOP OF METADATA
 {
-    "com.salesforce": "7.2.0",
```

Comment:
NB That version does not really matter (it's not touched by the setVersion.sh script currently).